Tools/TestWebKitAPI/Tests/WGSL/ConstLiteralTests.cpp

```diff
@@ -33,7 +33,7 @@
 static Expected<std::pair<WGSL::ShaderModule, WGSL::AST::Expression::Ref>, WGSL::Error> parseLCharPrimaryExpression(const String& input)
 {
     WGSL::ShaderModule shaderModule(input, { });
-    WGSL::Lexer<Latin1Character> lexer(input);
+    WGSL::Lexer<Latin1Character> lexer(input.span<Latin1Character>());
```

Comment:
Either it should just work, or with a deduction guide, you could probably get this to be:

```
WGSL::Lexer lexer(input.span<Latin1Character>());
```

Reply:
Success! I didn’t need to add a deduction guide